app/status.go

```diff
@@ -295,6 +295,45 @@ func (a *App) SetStatusAwayIfNeeded(userId string, manual bool) {
 	})
 }
 
+func (a *App) SetStatusDndIfNeeded(userId string, manual bool) {
```

Comment:
I think this should just be named `SetStatusDnd` (or better `SetStatusDoNotDisturb`) since it's guaranteed to set your status to dnd. `SetStatusAwayIfNeeded` is named as is because if some conditions aren't met it won't set you to away

Reply:
Also why the `manual` parameter? I think it's always going to be manual right?

Reply:
done